src/cli/bootupctl.rs

```diff
@@ -13,6 +13,10 @@ static SYSTEMD_ARGS_BOOTUPD: &[&str] = &[
     "PrivateNetwork=yes",
     "--property",
     "ProtectHome=yes",
+    // While only our main process during update catches SIGTERM, we don't
+    // want systemd to send it to other processes.
+    "--property",
+    "KillMode=mixed",
```

Comment:
Also needed in `contrib/packaging/bootloader-update.service` as if we already run under systemd we will not use systemd-run

Reply:
Thanks, fixed.